distribution/resources/cli-conf/logging.properties

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Year of the license header need to be changed

Reply:
Thanks. It is fixed now.